Better pause menu:

Jabka666 opened the pull request:
- Increased height of pause menu. Now it looks more proportionate;
- Fixed an inaccuracy showing a map seed instead of selected map name while playing on selected map;
- Made a capitalization of the buttons name which is more stylistically;
- Moved some ticks closer to each other in "Advanced" tab;
- Aligned buttons width;
- Fixed an inaccuracy showing the "Load Game" button while playing on "Perma Death" mode;
- Made a small spacing alignment between buttons;
- Added forgotten scale factor on the line #7701.
Initial commit without buttons name capitalization: [17203591f83dc92267bbcccf570e06c9fc579ca2](https://github.com/ChronoQuote/scpcb/commit/17203591f83dc92267bbcccf570e06c9fc579ca2)
Before:
![image](https://user-images.githubusercontent.com/41839972/186535319-f90cb36d-f7e7-4233-9398-39cfe4026ec2.png)
![image](https://user-images.githubusercontent.com/41839972/186535333-095d8043-5b10-44d3-a184-93bd92c45fb1.png)
After:
![image](https://user-images.githubusercontent.com/41839972/186535071-41286475-26b7-4533-b7a4-46f682d7edd4.png)
![image](https://user-images.githubusercontent.com/41839972/186535090-f1d94e83-4481-455c-b77f-f068c5dc5982.png)

Jabka666 reviewed (commented):
WARNING! Forgot to discard Ideal Parameters here!

ChronoQuote commented:
Seems to me like the boxes in the advanced tab were spaced apart as a way of grouping them into categories. Also, AA text looks isolated in both the before and after... maybe it should be grouped like normal and only be moved down while the framelimit slider is visible?

Jabka666 commented:
> Seems to me like the boxes in the advanced tab were spaced apart as a way of grouping them into categories. Also, AA text looks isolated in both the before and after... maybe it should be grouped like normal and only be moved down while the framelimit slider is visible?

Well yeah, would be better move AAText upper. It looks isolated because of Framelimit slidebar. Don't forget that slidebars don't have a rescale, so on lower resolutions it can be overlapped

Jabka666 commented:
- The ``AAText`` checkbox has been moved closer to the others;
- Added ``MenuGray`` texture for locked buttons, ticks, inputboxes and etc.;
- Added ``ButtonSFX2`` sound while clicling on locked button or tick;
- Added new parameter in ``DrawButton`` function called "locked" for easier adding locked buttons;
- Moved down arrows in save and custom map menus;
- Added ability to delete custom map;
- Made a capitalization of the buttons name which is more stylistically (in Main Menu);
- Reduce menu height when the ``Framelimit`` or ``Usertrack`` options are hidden;
- Reduced the number of saves per page from 6 to 5;
- Fixed being able to press the ``BACK`` button when deleting a save;
- Fixed arrows shifting one pixel in the save and custom map menus;
- Slightly improved input boxes design (used ``DrawFrame`` instead of ``Rect``, so they stylistically similar to buttons and ticks).
Before:
![image](https://user-images.githubusercontent.com/41839972/188268806-310dda02-bee3-48bd-b455-094fdac4b807.png)
![image](https://user-images.githubusercontent.com/41839972/188268817-9c884106-e67c-45f1-8ebb-a48a4c09d46b.png)
![image](https://user-images.githubusercontent.com/41839972/188268825-e350c433-b241-4cd3-8577-2c8bddb98ee1.png)
![image](https://user-images.githubusercontent.com/41839972/188268832-af2da83c-3ae6-489f-9000-5618d6c32297.png)
After:
![image](https://user-images.githubusercontent.com/41839972/188268938-bc1450e8-f6e9-48ed-adb3-27d0a4e9a50a.png)
![image](https://user-images.githubusercontent.com/41839972/188268946-41f52f81-06e1-4b11-9bc7-3b02f8dcc7be.png)
![image](https://user-images.githubusercontent.com/41839972/188268953-03380b7f-d2d4-4f26-8636-71b3243d8b86.png)
![image](https://user-images.githubusercontent.com/41839972/188268960-0149ccde-0e12-4ccf-8d7a-389e501e24ce.png)

Jabka666 commented:
- A "Back" wasn't all caps;
- A "Page" wasn't all caps;
- Two ``Screen gamma`` inscriptions didn't have a colon;
- ``Open/Close Inventory/Console`` was too long -> Changed to ``Toggle Inventory/Console``;
- Adjusted some frames by 1px;
- Added ``Enable launcher`` and ``Play startup videos`` ticks;
- Automatically turn off antialiasing option while playing in non-fullscreen mode. Also make the tick locked;
- Don't used ``True/False`` statements in ``options.ini`` anymore;
- Hide ``Open console on error`` option when the ``Enable console`` is turned off.
- Horizontally centered ``YES`` and ``NO`` deletion confirmation buttons;
- Better aligned ``ACHIEVEMENTS`` and ``MAIN MENU`` buttons on ending screen;
- Made achievement menu page buttons square and added disabled buttons to match load game/load map page buttons;
- Centered pause menu title text to match ending screen title text;
- Move user track mode text to right of tick box like it is in main menu;
- Declare ``TempStr`` as a local in menu drawing functions (and rename it to the more common ``StrTemp``);
- Center save file and custom map info text;
- Fixed SlideBar ``LOW`` and ``HIGH`` text being considerably lower than the bar.
Final results:
**NEW GAME:**
![image](https://user-images.githubusercontent.com/41839972/188288991-b33ccfc5-2c1f-4f58-860e-a69b226319c2.png)
**LOAD GAME:**
![image](https://user-images.githubusercontent.com/41839972/188288988-029a71a2-cecd-43de-b100-dc32d298677f.png)
**LOAD MAP:**
![image](https://user-images.githubusercontent.com/41839972/188288891-5b35a2da-3149-48aa-aa79-e8f59d48d009.png)
**GRAPHICS:**
![image](https://user-images.githubusercontent.com/41839972/188288904-ec7aa442-e314-452d-a2a3-150a2e4df22b.png)
**AUDIO:**
![image](https://user-images.githubusercontent.com/41839972/188288908-e0d02c06-40d7-4f1e-a5b3-0017ea6ec5fc.png)
**CONTROLS:**
![image](https://user-images.githubusercontent.com/41839972/188288919-5722881a-083d-4e82-a8e9-fa50cfb73186.png)
**ADVANCED:*
![image](https://user-images.githubusercontent.com/41839972/188288936-dcac08b2-8c1b-4fe6-811b-24d05b739253.png)

Jabka666 commented:
- Used ``ResetInput`` to prevent clicking on the locked button while quitting the game;
- Added ``HidePointer`` while loading a game.
